containers/aesmd/Dockerfile

```diff
@@ -19,25 +19,14 @@ RUN echo 'deb [arch=amd64] https://packages.microsoft.com/ubuntu/20.04/prod foca
 
 RUN apt-get update
 
-# TODO: Update Conclave to Intel SGX SDK 2.18 and remove package versions: https://r3-cev.atlassian.net/browse/CON-1243
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    sgx-aesm-service=2.17.100.3-focal1 \
-    libsgx-ae-id-enclave=1.14.100.3-focal1 \
-    libsgx-ae-le=2.17.100.3-focal1 \
-    libsgx-ae-pce=2.17.100.3-focal1 \
-    libsgx-ae-qe3=1.14.100.3-focal1 \
-    libsgx-ae-qve=1.14.100.3-focal1 \
-    libsgx-aesm-ecdsa-plugin=2.17.100.3-focal1 \
-    libsgx-aesm-launch-plugin=2.17.100.3-focal1 \
-    libsgx-aesm-pce-plugin=2.17.100.3-focal1 \
-    libsgx-aesm-quote-ex-plugin=2.17.100.3-focal1 \
-    libsgx-dcap-quote-verify=1.14.100.3-focal1 \
-    libsgx-enclave-common=2.17.100.3-focal1 \
-    libsgx-launch=2.17.100.3-focal1 \
-    libsgx-pce-logic=1.14.100.3-focal1 \
-    libsgx-qe3-logic=1.14.100.3-focal1 \
-    libsgx-urts=2.17.100.3-focal1 \
-    az-dcap-client
+    libsgx-quote-ex \
+    libsgx-dcap-ql \
+    az-dcap-client \
+    # We have demised EPID, so we don't want to install EPID related libraries
+    libsgx-ae-epid- \
+    libsgx-aesm-epid-plugin-
```

Comment:
@shamsasari I have found out this way of ignoring the packages for EPID, I think it is clean and it actually works

Reply:
Cool. Can you merge this in? I'd like to use this for the PyTorch sample.

Reply:
Perhaps the investigation in libsgx_urts.so.2 can be done separately.